lib/active_admin/helpers/collection.rb

```diff
@@ -7,7 +7,7 @@ module Collection
       # the ORDER statement mentions a column defined in the SELECT statement.
       #
       # We remove the ORDER statement to work around this issue.
-      def collection_size(collection=collection)
+      def collection_size(collection=collection())
```

Comment:
Do not use parentheses for method calls with no arguments.<br>Surrounding space missing in default value assignment.